keras/applications/imagenet_utils.py

```diff
@@ -89,17 +90,26 @@ def _obtain_input_shape(input_shape,
         data_format: image data format to use.
         include_top: whether the model is expected to
             be linked to a classifier via a Flatten layer.
+        weights: one of `None` (random initialization)
+            or "imagenet" (pre-training on ImageNet).
+            If weights="imagenet" input channels must be equal to 3.
```

Comment:
@Dref360 I actually copy that from the applications section. All CNN application have that same "imagenet" double-quote. That's why I did double-quote in the documentation.

I guess leave all double-quote or change all to single quotes. All the same to me.